packages/react-storage/src/components/StorageBrowser/views/LocationActionView/getActionViewTableData.ts

```diff
@@ -169,7 +169,11 @@ export const getActionViewTableData = <T extends TaskData = TaskData>({
               type: 'number',
               content: {
                 value: progress,
-                displayValue: `${getPercentValue(progress ?? 0)}%`,
+                displayValue: `${getPercentValue(
+                  // Zero-byte files are uploaded instantly and don't report a progress
```

Comment:
Would drop this line from the comment:
- a custom upload action may not exhibit this behavior
- `progress` may not always be upload specific

Reply:
Sounds reasonable to me. Will update to just include note that we're defaulting to 100% progress if value is unavailable but status is complete